lib.sh

```diff
@@ -282,6 +282,7 @@ input_box_flow() {
 }
 
 install_popup() {
+    msg_box "$SCRIPT_EXPLAINER"
```

Comment:
I am against this change. I think it is better to show it always at the beginning of the script no matter if you have it installed or not

Reply:
if you already installed it, you probably know what it is...